.gitignore

```diff
@@ -42,3 +42,7 @@ setup/win32/static/postgresql*.exe
 
 # avoid adding it after generation from tsconfig command
 tsconfig.json
+
+node_modules/
+package-lock.json
+package.json
```

Comment:
You probably shouldn't be touching this `.gitignore`, if this is because of the way you set up your javascript linter and you don't know how to fix it feel free to call me and I'll help you set it up so that this doesn't happen :)